### PR TITLE
Toc privacy cells

### PR DESCRIPTION
**Pull request by Rockindash**
Privacy Policy and Terms of Service cells added to Settings, Issue #1049

### CODE REVIEW

File: VultisigApp/VultisigApp/Utils/URL.swift [hunk 1]
```diff
@@ -12,4 +12,6 @@ class StaticURL {
     static let XVultisigURL = URL(string: "https://twitter.com/vultisig")!
     static let AppStoreVultisigURL = URL(string: "https://www.apple.com/")!
     static let DiscordVultisigURL = URL(string: "https://discord.gg/ngvW8tRRfB")!
+    static let PrivacyPolicyURL = URL(string: "https://docs.google.com/document/d/1gxA5OcCFGupWDyF3VCyKwwojA99CVngH9fLER8irRvI/edit#heading=h.1de5j5yfesa8")!
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Utils/URL.swift:15]: We need to wait for `vultisit.com` update the link to T&C , and Privacy Policy , then we can use the link

Comment (Rockindash) [reply VultisigApp/VultisigApp/Utils/URL.swift:15]: Ah ok, will draft PR until then.

Comment (realpaaao) [reply VultisigApp/VultisigApp/Utils/URL.swift:15]: @Rockindash see:
TOS: https://vultisig.com/termofservice
privacy: https://vultisig.com/privacy

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**